catwatch-backend/etc/scm/scm-source.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+REV=$(git rev-parse HEAD)
+URL=$(git config --get remote.origin.url)
+STATUS=$(git status --porcelain)
+if [ -n "$STATUS" ]; then
+    REV="$REV (locally modified)"
+fi
+# finally write hand-crafted JSON to scm-source.json
+echo '{"url": "git:'$URL'", "revision": "'$REV'", "author": "'$USER'", "status": "'$STATUS'"}' > target/generated-sources/scm-source.json
```

Comment:
you can replace the whole script with our provided Python CLI tool: http://docs.stups.io/en/latest/user-guide/application-development.html#docker